### PR TITLE
Reference pages for 'tile util' APIs - set 1

### PR DESCRIPTION
**Pull request by ganicke**
Put in some reference pages for the 'arcade-tile-util' APIs. This is just a partial set. I'll do a PR for the balance of them shortly.

Feel free to add/suggest more descriptive info/annotations.

RE: https://github.com/microsoft/pxt-arcade/pull/4691

### CODE REVIEW

File: docs/cover-all-tiles.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+controller.A.onEvent(ControllerButtonEvent.Pressed, function () {
+    tileUtil.coverAllTiles(assets.tile`myTile0`, assets.tile`myTile2`)
+    tileUtil.coverAllTiles(assets.tile`myTile1`, assets.tile`transparency16`)
+})
+tiles.setCurrentTilemap(tilemap`level1`)
+```
+
+## See also
+
+[replace all tiles](/tile-util/replace-all-tiles),
```
Comment (ganicke) [inline docs/cover-all-tiles.md]: Not sure how the see also paths will fit into the doc space. Might need to remove them if it doesn't work out.

File: docs/tilemap-property.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+```
+
+## Parameters
+
+* **data**: the tilemap to get a property for.
+* **prop**: the property to retrieve from the tilemap:
+>* `number of colunms`: the number of tile columns in the tilemap
+>* `number of rows`: the number of tile rows in the tilemap
+>* `height`: height in pixels of the tilemap
+>* `width`: width in pixels of the
```
Comment (riknoll) [inline docs/tilemap-property.md]:
```suggestion
>* `width`: width in pixels of the tilemap
```

File: docs/tilemap-property.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+* **prop**: the property to retrieve from the tilemap:
+>* `number of colunms`: the number of tile columns in the tilemap
+>* `number of rows`: the number of tile rows in the tilemap
+>* `height`: height in pixels of the tilemap
+>* `width`: width in pixels of the 
+>* `tile width`: the width in pixels of each tile location
+
+## Returns
+
+* a [number](/types/number) that is value of the requested property.
```
Comment (riknoll) [inline docs/tilemap-property.md]:
```suggestion
* a [number](/types/number) that is the value of the requested property.
```

**Review by riknoll — APPROVED** [review]
Looks good! There also needs to be entries for each of these markdown files in pxt.json but I can do that once this is merged

**Comment by ganicke** [comment]
@riknoll - you'll have to do the merge.

**Pull request merged**